client/src/raw.rs

```diff
@@ -473,6 +476,21 @@ macro_rules! proxier {
     };
 }
 
+macro_rules! namespaced_request {
+    ($req:ident) => {{
+        let ns_str = $req.get_ref().namespace.clone();
+        // Attach namespace header
+        $req.metadata_mut().insert(
+            TEMPORAL_NAMESPACE_HEADER_KEY,
+            ns_str
+                .parse()
+                .unwrap_or_else(|_| AsciiMetadataValue::from_static("")),
```

Comment:
Can you confirm that it's correct / what the reason is to insert an empty value in this case rather than omitting the header?

Reply:
I dunno that's it's well defined what would make sense to do here. The call _is supposed to be_ namespace specific, but, the name of it cannot be set as a header. So I suppose it makes sense to me to indicate it was trying to do a namespace call but couldn't fill the value.

Reply:
I think there should be a warning containing the value that could not be parsed.

Reply:
Sure, I can add that. But, I don't think it's actually possible since I doubt we would allow registering namespaces that have invalid chars.

Reply:
> I don't think it's actually possible
Yeah, a subset of impossible that maps to warn rather than panic, I'm thinking? Even if we knew we were talking to a server that wants this header, failing ATs / WFTs / WFs would be too strict I think. But I kind of don't think any subset of impossible maps to silent, except maybe some security situations.